app/models/manageiq/providers/vmware/infra_manager/inventory/parser/virtual_machine.rb

```diff
@@ -190,6 +190,7 @@ def parse_virtual_machine_disks(_vm, hardware, props)
         controller_type = case controller.class.wsdl_name
                           when /IDE/ then 'ide'
                           when /SIO/ then 'sio'
+                          when /AHCI/ then 'sata'
```

Comment:
Could you check for `/AHCI/, /SATA/`?  There are two classes of sata controller, [VirtualSATAController](https://code.vmware.com/apis/358/vsphere/doc/vim.vm.device.VirtualSATAController.html) and its subclass [VirtualAHCIController](https://code.vmware.com/apis/358/vsphere/doc/vim.vm.device.VirtualAHCIController.html)